content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,817 @@
+sent.
+
+The options in the :guilabel:`Winner Selection` field are as follows:
+
+- :guilabel:`Manual`: allows the user to determine the "winning" version of the mailing. This option
+  removes the :guilabel:`Send Final On` field.
+- :guilabel:`Highest Open Rate` (default): the mailing with the highest open rate is determined to
+  be the "winning" version.
+- :guilabel:`Highest Click Rate`: the mailing with the highest click rate is determined to be the
+  "winning" version.
+- :guilabel:`Highest Reply Rate`: the mailing with the highest reply rate is determined to be the
+  "winning" version.
+- :guilabel:`Leads`: the mailing with the most leads generated is determined to be the "winning"
+  version.
+- :guilabel:`Quotations`: the mailing with the most quotations generated is determined to be the
+  "winning" version.
+- :guilabel:`Revenues`: the mailing with the most revenue generated is determined to be the
+  "winning" version.
+
+The :guilabel:`Send Final On` field allows users to choose a date that is used to know *when* Odoo
+should determine the "winning" email, and subsequently, send that version of the email to the
+remaining recipients.
+
+.. image:: email_marketing/ab-test-tab.png
    :align: center
-   :alt: View of the building blocks used to create mailings in Odoo Email Marketing application.
+   :alt: View of the A/B Tests tab in Odoo Email Marketing application.
 
-.. tip::
-   To build an email from scratch, select the :guilabel:`Plain Text` template, and Odoo provides a
-   blank email canvas, which can be customized in a number of ways - either by using the front-end
-   rich text editor that accepts slash (:guilabel:`/`) commands, or with the XML code editor when
-   :ref:`Developer Mode (debug mode) <developer-mode>` is engaged, and the :guilabel:`</>` icon is
-   clicked.
+To the right of those fields is a :guilabel:`Create an Alternative Version` button. When clicked,
+Odoo presents a new :guilabel:`Mail Body` tab for the user to create an alternate version of the
+email to test.
+
+.. _email_marketing/settings_tab:
 
 Settings tab
 ------------
 
-To the right of the :guilabel:`Mail Body` tab is the :guilabel:`Settings` tab.
+The options present in the :guilabel:`Settings` tab of the mail form are divided into two sections:
+:guilabel:`Email Content` and :guilabel:`Tracking`.
 
 .. note::
-   The options available in the :guilabel:`Settings` tab will be different, depending on if the
-   :guilabel:`Mailing Campaigns` feature is activated in the :guilabel:`Settings` page of the
-   :menuselection:`Email Marketing` application (:menuselection:`Email Marketing --> Configuration
-   --> Settings`).
+   The options available in the :guilabel:`Settings` tab vary depending on if the :guilabel:`Mailing
+   Campaigns` feature is activated in the *Settings* page of the *Email Marketing* application
+   (:menuselection:`Email Marketing --> Configuration --> Settings`).
```

Comment:
No :guilabel: on "Mailing Campaigns" because the field is not on the screen yet. Can also simplify the navigation instructions and link to the **Mailing Campaigns** section :) 
```suggestion
   The options available in the :guilabel:`Settings` tab vary depending on if the *Mailing
   Campaigns* feature is activated in :menuselection:`Email Marketing --> Configuration -->
   Settings`. See :ref:`email_marketing/mailing-campaigns` for more information.
```